pycqed/analysis/analysis_toolbox.py

```diff
@@ -1352,13 +1353,13 @@ def color_plot_slices(xvals, yvals, zvals, ax=None,
             zvals[xx] = np.log(zvals[xx])/np.log(10)
 
     # add blocks to plot
-    hold = kw.pop('hold', False)
+    #hold = kw.pop('hold', False)
     for xx in range(len(xvals)):
         tempzvals = np.array([np.append(zvals[xx], np.array(0)),
                               np.append(zvals[xx], np.array(0))]).transpose()
-        im = ax.pcolor(xvertices[xx:xx+2],
-                       yvertices[xx],
-                       tempzvals, cmap=cmap)
+        # im = ax.pcolor(xvertices[xx:xx+2],
```

Comment:
Again, why is this commented?

Reply:
The lines that I commented out were reported by codacy as "possible new issues" because they were unused variables. They were in functions I have not looked at so I didn't want to completely remove these lines.